docs/develop/use-polyfills-to-support-ie11-in-your-office-add-in.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Use polyfills to support Internet Explorer 11 (IE11) in your Office Add-in
+description: Expand the reach of your Office Add-in to support the large number of customers still on IE11 by using polyfills. 
+ms.date: 05/13/2021
+localization_priority: Priority
+---
+# Use polyfills to support Internet Explorer 11 (IE11) in your Office Add-in
+
+There are still a large number of customers who use the Internet Explorer 11 (IE11) browser, and to reach them, you'll want your Office Add-in to successfully run on it. However, IE11 does not implement many JavaScript features that are available on modern browsers like Microsoft Edge and Google Chrome. This article shows how to use Polyfills in your Office Add-in to work around missing IE11 functionality.
+
+## How a polyfill works
+
+A polyfill is code that implements a feature on web browsers that do not support the feature. For example, in IE11, the `Number.isNan` function does not exist. If someone had code that needed to call `Number.isNan` they could write a polyfill to ensure their code is compatible with IE11. The following code sample shows how to implement a `Number.isNan` polyfill:
+
+```javascript
+if(!Number.isNan) { // If the function is not defined
+	// Write a definition for Number.isNan
+	Number.isNan = function(x) {
+		return typeof x === 'number' && x !== x;
+	}
+}
+```
+
+In the previous example, the code checks if `Number.isNan` exists. If it doesn’t, then we define an implementation for `Number.isNaN` that is used automatically whenever `Number.isNan` is invoked. If the programmer did not include this polyfill, their code would fail to run and throw an exception on certain outdated JavaScript engines.
```

Comment:
```suggestion
In the previous example, the code checks if `Number.isNan` exists. If it doesn’t, then you should define an implementation for `Number.isNaN` that is used automatically whenever `Number.isNan` is invoked. If you don't include this polyfill, your code will fail to run and throw an exception on certain outdated JavaScript engines.
```
Sorry, should have caught this in my first edit pass! Change to first person to be consistent with the rest of the article.